opencsp/common/lib/tool/log_tools.py

```diff
@@ -153,51 +149,22 @@ def multiprocessing_logger(log_dir_body_ext=None, level=log.INFO):
     return global_multiprocessing_logger
 
 
-def _add_stream_handlers(
-    logger: log.Logger, level: int, formatter: log.Formatter = None
-):
-    """Adds the stdout and stderr streams to the given logger.
+def _add_stream_handlers(logger_: log.Logger,
+                         level: int,
+                         formatter: log.Formatter = None) -> None:
+    """Adds streams to the given logger. Prints
 
     From https://stackoverflow.com/questions/16061641/python-logging-split-between-stdout-and-stderr
     """
     # stdout log
     h1 = log.StreamHandler(sys.stdout)
     h1.setLevel(log.DEBUG)
-    h1.addFilter(lambda record: record.levelno <= level)
+    h1.addFilter(lambda record: record.levelno >= level)
     h1.setFormatter(formatter)
-    logger.addHandler(h1)
-
-    # stderr log
-    h2 = log.StreamHandler(sys.stderr)
-    h2.setLevel(log.WARNING)
-    h2.setFormatter(formatter)
-    logger.addHandler(h2)

```

Comment:
I believe the error was caused by two things. 
1. This inequality was backwards
2. Having both stderr and stdout 

So I removed stderr and flipped the inequality.

Reply:
I disagree about removing stderr (see below for my reasoning why).

Looking at this code again, here's what I propose:

```python
    # stdout: log everything between level and warning
    h1 = log.StreamHandler(sys.stdout)
    h1.setLevel(level)
    h1.addFilter(lambda record: record.levelno < log.WARNING)
    h1.setFormatter(formatter)
    logger.addHandler(h1)

    # stderr: log everything warning and greater (warning, error, critical)
    h2 = log.StreamHandler(sys.stderr)
    h2.setLevel(log.WARNING)
    h2.setFormatter(formatter)
    logger.addHandler(h2)
```

Reply:
Reasons to keep stderr:
- Often times application integrators will limit output to only stderr as a first-pass filter for important information.
- Stderr also is implemented differently on most OSs. It isn't buffered, meaning it is more likely that the critical part of the log will be printed in the case that the program crashes halfway through printing the line.

Reply:
Oh, that makes a lot of sense!! Thanks for the code proposal, I think that's 100% correct. It seems to work now.